runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -579,7 +579,7 @@ func s:HandleEvaluate(msg)
 endfunc
 
 function! s:ShouldUseFloatWindow() abort
-  if has('nvim_open_win') && exists('g:termdebug_useFloatingHover') && (g:termdebug_useFloatingHover == 1)
+  if exists('*nvim_open_win') && exists('g:termdebug_useFloatingHover') && (g:termdebug_useFloatingHover == 1)
```

Comment:
while we are changing this, can use `get(g:, 'termdebug_useFloatingHover', 0)` to default to false. (though shouldn't we default to true? :)

Reply:
> though shouldn't we default to true? :)

+1

Reply:
b5b61d8